resources/lib/helper.py

```diff
@@ -72,24 +73,21 @@ def getVideoURL(json_obj):
   """
   video_url = None
   for video in json_obj["videoReferences"]:
-    if video["format"] == "hls":
-      alt = getAltUrl(video["url"])
-      if alt is None:
-        video_url = video["url"]
-      else:
-        video_url = alt
+    if video["format"] == "hls":            
+      if 'resolve' in video:
+         video_url = getResolvedUrl(video['resolve'])
+      if video_url is None:
+          video_url = video["url"]        
   return video_url
 
-def getAltUrl(video_url):
-  o = urlparse(video_url)
```

Comment:
If we don't need these methods from url lib, can you clean up the import at the top?

Reply:
good point, fixed